docs/faq.md

```diff
@@ -14,6 +14,20 @@ curl -fsSL https://ollama.com/install.sh | sh
 
 Review the [Troubleshooting](./troubleshooting.md) docs for more about using logs.
 
+## Is my GPU compatible with Ollama?
+
+### Nvidia
+Ollama supports Nvidia GPUs with compute capability 3.5 to 8.6.
+
+Check this table to see if your card is supported:
+[https://en.wikipedia.org/wiki/CUDA#GPUs_supported](https://en.wikipedia.org/wiki/CUDA#GPUs_supported)
```

Comment:
What do you think about including a link to this page which I've found super helpful as a decoder - https://developer.nvidia.com/cuda-gpus